fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -530,6 +534,20 @@ public void renameColumnNamePrefix(long idxId) {
         }
     }
 
+    public boolean isAbortDelete() {
+        boolean abortDelete = false;
+        if (keysType == keysType.PRIMARY_KEYS) {
+            for (Column col : getBaseSchema()) {
+                if (col.isAutoIncrement() && !col.isKey()) {
```

Comment:
If `delete where A=1 and B=2`, A is auto increment column, but not key; B is key?